### ISSUE TITLE
Licence 

### ISSUE THREAD

**Issue by Y-Taras** [kutlugsahin/ngx-smooth-dnd#21]
Hi, I would like to tell you that you did a great work! The most awesome thing animation.
But there's one problem/issue with license. I have not found any seperate license file nor license setting in your `package.json` file. 
So it would be nice to add it, so that people know how they are permitted to use it, and any restrictions you're placing on it.
 Thanks for considering my request.

**Comment by baleeds** [comment]
Agreed, I'd love to use this tool, but I only have access to MIT licensed tools.  Have you considered what license you would like to use?

**Comment by Y-Taras** [comment]
Yes, MIT would be fine.

**Comment by baleeds** [comment]
@kutlugsahin thoughts on MIT?
